config.py

```diff
@@ -127,6 +131,10 @@ class DevelopmentConfig(Config):
     PARTY_PASSWORD = os.getenv('PARTY_PASSWORD', 'secret')
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
 
+    NOTIFICATION_TEMPLATE_ID = os.getenv('NOTIFICATION_TEMPLATE_ID', "test_notification_template_id")
+
+    RM_NOTIFY_GATEWAY_URL = os.getenv('RM_NOTIFY_GATEWAY_URL', 'http://localhost:5000/email/')
```

Comment:
This is not the correct port for notify gateway. It should be 5181

Reply:
it s a dev configuration is not used ...by anyone. You cannot send any email from your local host. Is just for test purpose. Cahnged anyway.